joomla/joomla-cms
# [5.3] Upmerge changes from 5.2-dev 2024-10-29

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) merges the changes from 5.2-dev up into 5.3-dev which have been made since the last upmerge with PR #44348 , which are:
- PR #44092 
- PR #44349 
- PR #44166 
- PR #44129 
- PR #44222 
- PR #44340 
- PR #44368 

### Testing Instructions

Review. Check the commits of this PR and those in the 5.2-dev branch, and check the code changes in this PR.

### Link to documentations
Please select:
